src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx

```diff
@@ -6702,10 +6702,6 @@ The Query Processor estimates that implementing the following index could improv
     <value>ON</value>

```

Comment:
There's a number of other values here that look like they shouldn't be localized. In general - if the value is passed as an option to the T-SQL (which most options are) then it should NOT be localized. 

Could you make a work item to follow up on verifying that you're handling these correctly? An easy way to check is by changing the name here in the resx and then trying to update options using these values. That should still work fine since these should be display names only.

Reply:
Created a task https://github.com/microsoft/azuredatastudio/issues/24084!